src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5493,7 +5493,7 @@ Unify Enum declarations with other block type declarations.
         <native-equiv name="Expression.Opcodes.CollectionInitializer" />
       </node-kind>
 
-      <child name="Type" kind="NonArrayTypeName">
+      <child name="Type" kind="NonArrayNonTupleTypeName">
```

Comment:
>NonArrayNonTupleTypeName [](start = 31, length = 24)

It looks like parser accepts tuple type syntax at this position, but reports an error. It is not clear what is the expected behavior of the factory methods should be in this case. Perhaps this is a question for the Public API Design Committee. What does C# do?

Reply:
Perhaps the error should be moved to semantic analysis.